python/interpret_community/dataset/dataset_wrapper.py

```diff
@@ -215,6 +294,25 @@ def one_hot_encode(self, columns):
         self._dataset = self._one_hot_encoder.fit_transform(self._dataset)
         return self._one_hot_encoder
 
+    def timestamp_featurizer(self):
+        """Featurizes the timestamp columns.
+
+        :return: The transformation steps to featurize the timestamp columns.
+        :rtype: DatasetWrapper
+        """
+        if self._timestamp_featurized:
+            return self._timestamp_featurizer
+        # Optimization so we don't redo this operation multiple times on the same dataset
+        self._timestamp_featurized = True
```

Comment:
nit: I'd prefer to set this to True just before the function returns (eliminate possible confusion when debugging this function)

Reply:
nvm I see the return right below